cmd/docker/docker.go

```diff
@@ -47,10 +47,15 @@ func newDockerCommand(dockerCli *command.DockerCli) *cli.TopLevelCommand {
 		},
 		Version:               fmt.Sprintf("%s, build %s", version.Version, version.GitCommit),
 		DisableFlagsInUseLine: true,
+		CompletionOptions: cobra.CompletionOptions{
+			DisableDefaultCmd:   false,
+			HiddenDefaultCmd:    true,
+			DisableDescriptions: true,
```

Comment:
Out of curiosity, you don't want descriptions?
I notice that the current docker completion system does provide descriptions.

Reply:
Some description are too long and break, haven't check the actual constraint to propose a fix